bin/ghe-restore

```diff
@@ -45,6 +46,10 @@ while true; do
       restore_settings=true
       shift
       ;;
+    --skip-pages)
+      export GHE_SKIP_PAGES=true
```

Comment:
Is this variable used outside of this script? That is, does it need to actually be exported?

Reply:
We elected to follow the existing pattern used inside the script as we weren't initially sure if we'd have to pass the value through to a child script or set it via config (see @jianghao0718's comment above).

Reply:
the script has both local variables and exported variables. it seems like if it's contained to just this script, we should go with a local variable

Reply:
but yeah once we land on flag vs config file, that will say which method works

Reply:
After discussion we've opted to use config settings instead of CLI flags